package.json

```diff
@@ -62,7 +62,14 @@
     "@47ng/check-env": "^2.0.1",
     "@babel/core": "^7.11.6",
     "@babel/generator": "^7.11.6",
+    "@babel/helper-compilation-targets": "^7.10.4",
```

Comment:
`next build` is failing because of these missing dependencies. What are we doing wrong? (there are still more 😱)